org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,100 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--device=all",
+    "--filesystem=xdg-documents",
+    "--filesystem=xdg-download"
+  ],
+  "modules": [
+    "shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json",
+    {
+      "name": "cuda-toolkit",
+      "only-arches": [
+        "x86_64"
+      ],
+      "cleanup": [
+        "/cuda"
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir /app/cuda",
+        "sh cuda_toolkit.run --silent --toolkit --toolkitpath=/app/cuda",
```

Comment:
This will extract 7 GiB of files while your app only seems to require 3 libraries.

Please consider downloading these libraries in [standalone mode](https://developer.download.nvidia.com/compute/cuda/redist/). That will help speed up the build time considerably while also avoid wasting disk space network bandwidth.

Reply:
mmm... in the standalone mode I don't see the needed libnvrtc, where did you see it? thanks for the suggestion, reducing the space would be much appreciated.

Reply:
https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvrtc/

Reply:
Man, thank you so much. you reduced my build size by 5 times.